app/src/main/java/com/borconi/emil/wifilauncherforhur/receivers/WifiReceiver.java

```diff
@@ -0,0 +1,29 @@
+package com.borconi.emil.wifilauncherforhur.receivers;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.util.Log;
+
+import com.borconi.emil.wifilauncherforhur.listeners.WifiListener;
+
+public class WifiReceiver extends BroadcastReceiver {
```

Comment:
dunno if I can remove this file (don't know if it works for Android > 9). Leave it for now.